extensions/dacpac/src/test/testContext.ts

```diff
@@ -102,7 +104,13 @@ export function createViewContext(): ViewTestContext {
 
 	let buttonBuilder: azdata.ComponentBuilder<azdata.ButtonComponent> = {
 		component: () => button,
-		withProperties: () => buttonBuilder,
+		withProperties: (properties) => {
```

Comment:
you didnt need to change button builder to function? its great if this alone worked - ignore the comment if so. Just a caution that I noticed (for radio button) that was re-updating the same builder every time. So say it had simple click for a non '...' button. Then next time weget a buttin with '...' and change the onDidclick - it was still the same button object that changed so OnDidclick was updated for both buttons.